optional_plugins/varianter_yaml_to_mux/avocado_varianter_yaml_to_mux/__init__.py

```diff
@@ -466,6 +467,13 @@ class YamlToMux(mux.MuxPlugin, Varianter):
     description = 'Multiplexer plugin to parse yaml files to params'
 
     def initialize(self, config):
+        """
+        :warning message: We use `ast.literal_eval()` despite its warning.
```

Comment:
:warning message: should not be part of the summary description. See PEP257. Also, not sure if it is a good idea to have a possible "security" branch annouced on our docstring.

Reply:
@beraldoleal Can you suggest or help me with arriving at the correct docstring here please ?

Reply:
@beraldoleal Any suggestions here please ?

Reply:
A simple message with something like: "This method is doing ... ..." in one line will be fine here.

Reply:
@beraldoleal So, we do not want warnings ?

Reply:
@narasimhan-v This is a security issue, right? Should we announce security failures so publicly?

Reply:
Got it. I added a warning because it was warn in the ast library. But agreed that we need not announce this.
If there is no need to warn, then should I remove the docstring ? There was none previously.

Reply:
I'm ok with that.

Reply:
Pushed the latest change.

Reply:
> :warning message: should not be part of the summary description. See PEP257. Also, not sure if it is a good idea to have a possible "security" branch annouced on our docstring.

`ast.literal_eval` is not ideal indeed, but I think is the best we can do here at this time. A few points:

1. The risk potential is "only" to crash the Python interpreter, and not to inject code or escape some kind of isolation
2. This happens early in the job initialization, so if a crash happens users will know about it
3. I think it'd be really hard to provide a "sufficiently large/complex" string to crash it
4. We already use that in the safeloader module anyway.